config/locale/de.json

```diff
@@ -1,39 +1,204 @@
+	"webhook":"webhook",
+	"add":"hinzufügen",
+	"<Your-Webhook-url>":"<Webhook-URL>",
+	"channel":"kanal",
+	"no valid areas there, please use one of":"Fehler: Kein gültiges Gebiet angegeben. Folgende Gebiete sind verfügbar:",
+	"Added areas:":"Hinzgefügte Gebiete:",
+	"Removed areas:":"Entfernte Gebiete",
+	"Current configured areas are:":"Aktuell sind folgende Gebiete festgelegt:",
+	"nameYourWebhookName":"webhookName",
+	"remove":"entfernen",
+	"level":"level",
+	"distance":"Entfernung",
+	"raids":"Raids",
+	"controlled by":"kontrolliert von",
+	"must be an EX Gym":"muss eine Ex-Raid-Arena sein",
+	"eggs":"Eier",
+	"reward":"Belohnung",
+	"mega energy":"Mega-Energie",
+	"Mega Energy":"Mega-Energie",
+	"energy":"Energie",
+	"or more stardust":"oder mehr Sternenstaub",
+	"template":"vorlage",
+	"d":"d",
+	"404 No raid egg levels found":"Fehler: Kein Raid-Ei-Level angegeben.",
+	"female":"weiblich",
+	"male":"männlich",
+	"genderless":"geschlechtunbekannt",
+	"clean":"aufräumen",
+	"404 No valid invasion types found":"Fehler: Es wurde kein gültiger Typ einer Team-GO-Rocket-Invasion angegeben.",
+	"New location":"Neuer Standort",
+	"I set ":"der Standort von „",
+	"'s location to the following coordinates in":"“ wurde wie folgt gesetzt:",
+	"Oops, you need to specify more than just a city name to locate accurately your position":"Fehler: Der angegebene Standort konnte nicht eindeutig bestimmt oder gefunden werden.",
+	"gen":"gen",
+	"all items":"alle items",
+	"all pokemon":"alle pokemon",
+	"stardust":"sternenstaub",
+	"Stardust":"Sternenstaub",
+	"404 No valid quests found":"Fehler: Kein gültiges Quest angegeben.",
+	"form":"form",
+	"404 no valid tracks found":"Fehler: Kein gültiger Raid-Boss oder gültiges Raid-Level angegeben.",
+	"everything":"alle",
+	"individually":"individuell",
+	"maxlevel":"maxlevel",
+	"great":"super",
+	"greatcp":"superwp",
+	"ultra":"hyper",
+	"ultracp":"hyperwp",
+	"maxcp":"maxwp",
+	"maxiv":"maxiv",
+	"maxweight":"maxgewicht",
+	"maxatk":"maxatk",
+	"maxdef":"maxdef",
+	"maxsta":"maxsta",
+	"cp":"wp",
+	"iv":"iv",
+	"atk":"atk",
+	"def":"def",
+	"sta":"sta",
+	"weight":"gewicht",
+	"404 No monsters found":"Fehler: Kein gültiges Pokémon angegeben.",
+	"Your location is currently set to":"Folgender Standort ist gesetzt:",
+	"You have not set a location yet":"Es wurde noch kein Standort gesetzt.",
+	"Your alerts are currently":"Benachrichtigungen sind aktuell",
+	"enabled":"aktiviert.",
+	"disabled":"deaktiviert.",
+	"You are currently set to receive alarms in":"Folgende Gebiete sind gesetzt:",
+	"You have not selected any area yet":"Es wurde kein Gebiet gesetzt.",
+	"You're tracking the following monsters:":"Folgende Pokémon werden beobachtet:",
+	"You're not tracking any monsters":"Es werden keine Pokémon beobachtet.",
+	"Everything":"_alle_",
+	"any":"_alle_",
+	"greatpvp":"Superliga",
+	"ultrapvp":"Hyperliga",
+	"gender":"Geschlecht",
```

Comment:
this is a command parameter, so should be lower case

Reply:
Sure about that? All 4 seem to be only used in the output of /tracking ...?